Tiltfile

```diff
@@ -224,6 +236,7 @@ k8s_resource(
     new_name="grafana",
     workload="observability",
     extra_pod_selectors=[{"app.kubernetes.io/name": "grafana"}],
+    # DEV_NOTE: We're 3071 instead of Grafana's default 3000 to avoid conflicts with other Grafana instances
```

Comment:
Shouldn't this be `3069`?

Reply:
This should be deleted since I didn't get it working right away so I moved on .

Reply:
Also, with the new default port of PATH begin `3069`, we can use the default Grafana port.